internal/config/config.go

```diff
@@ -40,6 +64,30 @@ type OperatorCfg struct {
 	LogLevel string
 	// DevMode when set to "true" logs in unstructured text instead of JSON.
 	DevMode bool
+	// DefaultOriginDomain represents a valid domain to define in default origin.
+	DefaultOriginDomain string
+	// CloudFrontPriceClass determines how many edge locations CloudFront will use for your distribution.
+	// ref: https://docs.aws.amazon.com/cdk/api/latest/typescript/api/aws-cloudfront/priceclass.html
+	CloudFrontPriceClass string
```

Comment:
Looks like you got the enum values from the TypeScript SDK hehe

For golang this should be `PriceClass_All`. Try Ctrl+F "PriceClassPriceClassAll" [here](https://docs.aws.amazon.com/sdk-for-go/api/service/cloudfront/)

Needs changing in the viper default and .env.example